frontend/src/scenes/teamActivityDescriber.tsx

```diff
@@ -366,6 +366,58 @@ const teamActionsMapping: Record<
             ],
         }
     },
+    revenue_tracking_config: (change): ChangeMapping | null => {
+        if (!change) {
+            return null
+        }
+        const beforeEvents: RevenueTrackingEventItem[] =
+            typeof change.before === 'object' && change.before && 'events' in change.before ? change.before.events : []
+        const afterEvents: RevenueTrackingEventItem[] =
+            typeof change.after === 'object' && change.after && 'events' in change.after ? change.after.events : []
+
+        const beforeEventNames = beforeEvents?.map((event) => event?.eventName)
+        const afterEventNames = afterEvents?.map((event) => event?.eventName)
+        const addedEvents = afterEventNames?.filter((event) => !beforeEventNames?.includes(event))
+        const removedEvents = beforeEventNames?.filter((event) => !afterEventNames?.includes(event))
+        const modifiedEvents = afterEventNames?.filter((event) => beforeEventNames?.includes(event))
+
+        return {
+            description: [
+                <>
+                    {interleaveArray(
```

Comment:
I don't think you need this?

First, it looks wrong, because you'll add a comma after the first entry which will make the text read like `"Updated revenue tracking config: , added 2 events, removed 3 events"`, so you at least need to remove the first part of the array from inside `interleaveArray`

Second, isn't this as simple as `.join(", ")`? They're all strings, they're not `JSX.Element`s

Reply:
Ah yeah, you're right, this changed a bit over time, at one point they were JSX.Elements